bin/update-news

```diff
@@ -0,0 +1,107 @@
+#! /usr/bin/env python
+"""
+This is a script to help us keep our "blog/news.json" file up to
+date. It will scan the "blog" directory, sort items by date (which
+is just title) and then read the metadata from the entries to update
+the "news.json" file.
+"""
+from __future__ import annotations
+
+import datetime
+import json
+import os
+import re
+import sys
+from typing import Sequence, Any, Iterator
+
+try:
+    import yaml
+except ImportError:
+    print("Please install PyYaml to use this script")
```

Comment:
The rest of the community is using `ruamel_yaml` for a while, so maybe we should rely on that one. Chances are it's already installed. I wouldn't silent the exception too either. Maybe there's an underlying issue (like a misplaced shared library).